qa/workunits/ceph-helpers.sh

```diff
@@ -1607,6 +1649,19 @@ function test_wait_background() {
     if [ ! -z "$pids" ]; then return 1; fi
 }
 
+function get_admin_socket()
+{
```

Comment:
Most ceph-helper functions have a **dir** first argument and work relative to it. It's goot to have a get_admin_socket so the caller does not have to hardwire $dir/$client.asok.

<pre>
function get_admin_socket() {
  local dir=$1
  local client=$1
..
</pre>